core/src/main/scala/cats/ContravariantMonoidal.scala

```diff
@@ -38,7 +38,7 @@ trait ContravariantMonoidal[F[_]] extends ContravariantSemigroupal[F] with Invar
    * the diagonal
    */
   def trivial[A]: F[A] = contramap(unit)(_ => ())
-
+  def conquer[A]: F[A] = trivial[A]
```

Comment:
why do we need this alias? Can we remove it?

Reply:
This is here primarily in deference to those Haskell users used to the original text - in that text `conquer` figures as a somewhat more primary combinator than in ours because they lack the `Semiringal` hierarchy that we have in this library.

Reply:
Got it, yeah I think there was a fair amount of discussion about whether to keep these names in earlier versions of the diff as well, and some folks explicitly wanted the aliases so they had familiar names. I can see the perspective you're putting forward here though as well, so I'll go ahead and nix these aliases.